CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## [_Unreleased_](https://github.com/thoughtbot/yesod-auth-oauth2/compare/v0.7.0.2...main)
+## [_Unreleased_](https://github.com/thoughtbot/yesod-auth-oauth2/compare/v0.7.0.3...main)
```

Comment:
I'd either remove this (and version bump in its own PR), or include the `package.yaml` / `.cabal` change to `version` here too.